### ISSUE TITLE
Aria-disabled=true is not defined for the disabled 'Clear column filters' control and hence screen reader is not announcing it as disabled.

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7235]
### Description: ###
Screen reader users will get impacted if the screen reader does not announce the state as 'disabled' for the disabled control. Screen reader users may not know that it is a disabled control and hence might try accessing it but end up in no action and disappointment.

### Test Environment: ###
OS Build: Version 21H2(22000.613)
Browser: Chrome Version 100.0.4896.127 (Official Build) (64-bit) 
Screen reader: Non-Visual Desktop Access (NVDA) (2021.1)

### Repro steps: ###
1.Open URL: https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel-style-filtering
2. Grids excel style filtering pen page will open.
3. Reach to the main landmark content where ' Examples' tab is selected.
4. With 6-tab press, reach to the column header of the available table.
5. Press ctrl + shift + L, the context menu opens.
6. Verify the issue that 'clear' control is defined with 'aria-disabled' property or not.

### Actual results: ###
Aria-disabled=true is not defined for the disabled 'Clear column filters' control and hence screen reader is not announcing it as disabled. The keyboard focus is moving to the disabled control and screen readers are not announcing it as a disabled control.

### Expected results: ###
Aria-disabled=true should be defined for the disabled 'Clear column filters' control and hence screen reader can announce it as disabled.

### Additional notes: ###
Same Issue Repro with Narrator and JAWS.
Same Issue Reproes with the same control in 'Name filter' present throughout the application.

### Attachments: ###
<img width="959" alt="MAS1 3 1_aria-disabled not announced" src="https://user-images.githubusercontent.com/93735775/183306247-ca30914b-9e3c-4619-9c5a-dd7be7b81367.png">

https://user-images.githubusercontent.com/93735775/183306263-edf11271-d308-4ce7-96e0-179acc09441b.mp4

**Comment by mportuga** [comment]
This defect is not for UI-Grid, but for Ignite UI, which is managed by a completely different team.
